templates/events/event-edit.html

```diff
@@ -977,7 +977,7 @@ <h5>Display Only with Super Event</h5>
                             <div class = "form-group row" >
                                   <div class="form-group col-sm-3 offset-sm-2">
                                         {% if sub.name %}
-                                            <input class="form-control search-id" id="sub-id" name="name" type="text" placeholder="Enter a keyword of subevent title, and select the subevent from the dropdown list" value="{{sub.name}}" />
+                                            <input class="form-control search-id" id="sub-id" name="name" type="text" placeholder="Search sub-events by words in the title." value="{{sub.name}}" />
```

Comment:
The sub-events placeholder text is in multiple places. The value has to be changed to "Search sub-events by title."

Reply:
I checked again. I did not find other places. Can you take another look?

Reply:
Provided additional in-line suggestions. Resolving this comment.